src/assets/scss/foundations.scss

```
@@ -208,6 +212,11 @@ table {
 		background-color: var(--lightest-background-color);
 		padding: 0.25rem 0.5rem;
 	}
+
+	@media screen and (max-width: 768px) {
+		display: block;
```

```suggestion
		display: block;
```
I came across this approach as well. Using display: block for a `<table>` tag will change its display behaviour.  When we set display: block, the `<table>` element will behave like a block-level element instead of a table. I'm not sure if this is the best approach, so I'll check for any potential side effects.

---

Thanks. I think we can fix that pretty easily using CSS.